grafana-plugin/src/models/user/user.ts

```diff
@@ -110,13 +116,17 @@ export class UserStore extends BaseStore {
     delete this.itemsCurrentlyUpdating[userPk];
   }
 
-  @action
-  async updateItems(f: any = { searchTerm: '' }, page = 1, invalidateFn?: () => boolean): Promise<any> {
+  async search<UT = User>(f: any = { searchTerm: '' }, page = 1): Promise<PaginatedUsersResponse<UT>> {
     const filters = typeof f === 'string' ? { searchTerm: f } : f; // for GSelect compatibility
     const { searchTerm: search, ...restFilters } = filters;
-    const response = await makeRequest(this.path, {
+    return makeRequest<PaginatedUsersResponse<UT>>(this.path, {
       params: { search, page, ...restFilters },
     });
```

Comment:
added to use [here](https://github.com/grafana/oncall/pull/3241/files#diff-fef683d27c367f37f7bd5d81805c341bf75050886667a48077f6d1afe58f7f67R116-R117). Also, see comment [here](https://github.com/grafana/oncall/pull/3241/files#r1380188290).